src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx

```diff
@@ -159,24 +160,34 @@ class ImportAuthenticatorScreen extends Component<Props, State> {
     return !!(name && mnemonic && !this.hasErrors());
   };
 
+  sendFeedback = () => {
+    const { name } = this.state;
+    if (!!!name.trim()) {
```

Comment:
I think that there's no difference between `!` and `!!!`

Reply:
yea id duplicate from code on top :)